src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -229,6 +229,9 @@ private function findServiceIdsContaining(ContainerBuilder $builder, $name)
         $serviceIds = $builder->getServiceIds();
         $foundServiceIds = array();
         foreach ($serviceIds as $serviceId) {
+            if (0 === strpos($serviceId, '.') && 0 !== strpos($name, '.')) {
+                continue;
+            }
```

Comment:
shouldn't we keep listing them when `--show-hidden` is passed?

Reply:
indeed, fixe thanks.